homeassistant/components/life360/config_flow.py

```diff
@@ -1,108 +1,190 @@
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle a config flow initiated by the user."""
+        if not user_input:
+            return self.async_show_form(
+                step_id="user", data_schema=vol.Schema(account_schema())
+            )
+
+        self._username = user_input[CONF_USERNAME]
+        self._password = user_input[CONF_PASSWORD]
+
+        await self.async_set_unique_id(self._username.lower())
+        self._abort_if_unique_id_configured()
+
+        return await self._async_verify("user")
+
+    async def async_step_reauth(self, user_input: dict[str, Any]) -> FlowResult:
+        """Handle reauthorization."""
+        self._username = user_input[CONF_USERNAME]
+        self._reauth_entry = self.hass.config_entries.async_get_entry(
+            self.context["entry_id"]
         )
+        # Always start with current credentials since they may still be valid and a
+        # simple reauthorization will be successful.
+        return await self.async_step_reauth_confirm(user_input)
+
+    async def async_step_reauth_confirm(self, user_input: dict[str, Any]) -> FlowResult:
+        """Handle reauthorization completion."""
+        self._password = user_input[CONF_PASSWORD]
+        return await self._async_verify("reauth_confirm")
+
+
+class Life360OptionsFlow(OptionsFlow):
+    """Life360 integration options flow."""
+
+    def __init__(self, config_entry: ConfigEntry) -> None:
+        """Initialize."""
+        self.config_entry = config_entry
+
+    async def async_step_init(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle account options."""
+        options = self.config_entry.options
+
+        if user_input is not None:
+            new_options = _extract_account_options(user_input)
+            return self.async_create_entry(title="", data=new_options)
+
+        return self.async_show_form(
+            step_id="init", data_schema=vol.Schema(_account_options_schema(options))
+        )
+
+
+def _account_options_schema(options: Mapping[str, Any]) -> dict[vol.Marker, Any]:
+    """Create schema for account options form."""
+    def_limit_gps_acc = options[CONF_MAX_GPS_ACCURACY] is not None
+    def_max_gps = options[CONF_MAX_GPS_ACCURACY] or vol.UNDEFINED
+    def_set_drive_speed = options[CONF_DRIVING_SPEED] is not None
+    def_speed = options[CONF_DRIVING_SPEED] or vol.UNDEFINED
+    def_scan_interval = options[CONF_SCAN_INTERVAL]
+    def_show_driving = options[SHOW_DRIVING]
+
+    return {
+        vol.Required("limit_gps_acc", default=def_limit_gps_acc): bool,
+        vol.Optional(CONF_MAX_GPS_ACCURACY, default=def_max_gps): vol.Coerce(float),
+        vol.Required("set_drive_speed", default=def_set_drive_speed): bool,
+        vol.Optional(CONF_DRIVING_SPEED, default=def_speed): vol.Coerce(float),
+        vol.Optional(CONF_SCAN_INTERVAL, default=def_scan_interval): vol.Coerce(float),
```

Comment:
We don't allow scan interval as option. Every integration that supports config entries allows the user to turn off automatic polling and then the user can automate the update interval as needed with the entity service `homeassistant.update_entity`.

Reply:
Ok.